spec/spec_helper.rb

```diff
@@ -8,10 +8,11 @@
 require 'redis'
 require 'rollout'
 
-Redis.current = Redis.new(
+REDIS_CURRENT = Redis.new(
   host: ENV.fetch('REDIS_HOST', '127.0.0.1'),
   port: ENV.fetch('REDIS_PORT', '6379'),
   db: ENV.fetch('REDIS_DB', '7'),
+  reconnect_attempts: [0.05, 0.1, 0.2]
```

Comment:
```
$ bundle exec rspec

An error occurred while loading ./spec/rollout/feature_spec.rb.
Failure/Error:
  REDIS_CURRENT = Redis.new(
    host: ENV.fetch('REDIS_HOST', '127.0.0.1'),
    port: ENV.fetch('REDIS_PORT', '6379'),
    db: ENV.fetch('REDIS_DB', '7'),
    reconnect_attempts: [0.05, 0.1, 0.2]
  )

NoMethodError:
  undefined method `to_i' for [0.05, 0.1, 0.2]:Array
# …/.gem/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:497:in `_parse_options'
# …/.gem/ruby/3.2.0/gems/redis-4.8.1/lib/redis/client.rb:92:in `initialize'
# …/.gem/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:87:in `new'
# …/.gem/ruby/3.2.0/gems/redis-4.8.1/lib/redis.rb:87:in `initialize'
# ./spec/spec_helper.rb:11:in `new'
# ./spec/spec_helper.rb:11:in `<top (required)>'
# ./spec/rollout/feature_spec.rb:1:in `require'
# ./spec/rollout/feature_spec.rb:1:in `<top (required)>'
```

Looks like this `reconnect_attempts` is broken for `redis-4.8.1` which is in within the `>= 4.0', '< 6.0` declared dependency.